aws/modules/deploy/helms/01-gpii-preferences/values.yaml

```diff
@@ -17,7 +17,7 @@ issuerRef:
   name: letsencrypt-production
 
 dnsNames:
-- preferences.stg.gpii.net
+- preferences.test.local
```

Comment:
This is a better default than preferences.stg.gpii.net, so that's good.

Can you clarify what this variable is used for?

Reply:
This is the list of host names for nginx-ingress. It is in the README.

Reply:
Aha, great!